src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/legacy/controller/LegacyController.kt

```diff
@@ -110,7 +110,7 @@ class LegacyController(
     @RequestBody adjustment: LegacyAdjustment,
   ) {
     legacyService.update(adjustmentId, adjustment).also {
-      eventService.update(adjustmentId, adjustment.offenderNo, AdjustmentSource.NOMIS)
+      eventService.update(adjustmentId, adjustment.offenderNo, AdjustmentSource.NOMIS, legacyService.transform(adjustment.adjustmentType))
```

Comment:
Can you revert this change. We don't need to pass the adjustment type into the event service here.

Reply:
I've reverted this change